src/inference/src/dev/threading/thread_affinity.cpp

```diff
@@ -119,7 +119,10 @@ std::tuple<CpuSet, int> get_process_mask() {
     }
     return std::make_tuple(nullptr, 0);
 }
-void release_process_mask(cpu_set_t*) {}
+void release_process_mask(cpu_set_t* mask) {
+    if (nullptr != mask)
+        CPU_FREE(mask);
```

Comment:
why use CPU_FREE here, allocate instance is using `new DWORD_PTR(pro_mask)` in line 117

Reply:
I  google CPU_FREE, it seem it's a function  on linux https://manpath.be/c8/3/CPU_ZERO

Reply:
Ok, changed to `delete`.